### ISSUE TITLE
Do we have a permission from Strava?

### ISSUE THREAD

**Issue by Spanholz** [allier-osm/strava_heatmap#3]
Do we have the official permission from Strava somewhere to use the Heatmap? According to @bhousel here: https://www.reddit.com/r/openstreetmap/comments/8y0tk5/strava_heatmap_as_custom_background_in_id/e28lfob/ we are not allowed to use Stravas Heatmap outside of their own iD fork. 

This also seems the case in the official OSM layer index: https://github.com/osmlab/editor-layer-index/issues/294

**Comment by allier-osm** [comment]
Please see https://josm.openstreetmap.de/ticket/16100 for details on this topic, including licensing.

**Comment by Spanholz** [comment]
Sorry, but [this](https://github.com/osmlab/editor-layer-index/pull/373#issuecomment-352544421) is no real yes from Strava. We have nothing in our hands as long as their is no official agreement from Strava. 

I would love to use the Heatmap but currently we can't.

**Comment by allier-osm** [comment]
@don-vip Do you have any idea about this? That would mean that the Strava heatmaps should be removed altogether from the default JOSM imagery layers...

**Comment by don-vip** [comment]
Sorry, no idea. Best is to contact Strava directly.

**Comment by allier-osm** [comment]
OK, then I will contact Strava.
I will not publish the plugin in the plugins list until I get a confirmation from them.

**Comment by StevenClays** [comment]
Any news on Strava's permission?

**Comment by solitone** [comment]
Still no news?

**Comment by pgavrilov** [comment]
https://twitter.com/paulmach/status/455182880306905088
Guillaume Rischard: «OK to use heatmap in JOSM editor?»
Paul Mach, Former Director of Strava Labs: «Feel free to use the heatmap tiles for any map editing»
13 apr 2014

**Comment by govvin** [comment]
Another user recently got in touch with Strava through email and they responded positively to the request. They posted the email response here: 
https://openstreetmap.org/wiki/Permissions/Strava

It falls short of a formal waiver, but it confirmed a practice that's been previously allowed.

**Comment by solitone** [comment]
That's interesting, but why don't they state it officially?

**Comment by govvin** [comment]
I agree, and we're also wondering about the same, and whether they can
convince Strava to issue a more formal letter.

**Comment by talllguy** [comment]
[Here's where the permission was given](https://lists.openstreetmap.org/pipermail/talk/2019-November/083564.html).
